packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -719,8 +719,7 @@
     }
     &__text {
         position: absolute;
-        color: var(--status-success);
+        color: var(--status-success) !important;
```

Comment:
both `Text` & `ContractCard` components are in `components` package and the css for the package giving precedence to Text component styles